packages/discovery-provider/ddl/migrations/0138_add_playlist_trending_scores.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS public.playlist_trending_scores (
+    playlist_id integer NOT NULL,
+    type character varying NOT NULL,
+    version character varying NOT NULL,
+    time_range character varying NOT NULL,
+    score double precision NOT NULL,
+    created_at timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+DO $$
```

Comment:
odd... why is this not just in the CREATE TABLE statement 

```
    PRIMARY KEY (playlist_id, type, version, time_range)
```

Reply:
Because I am a n00b.